src/term/mod.rs

```diff
@@ -228,6 +228,9 @@ pub struct Term {
     /// Alt cursor
     alt_cursor: Point,
 
+    /// Saved cursor position
+    saved_cursor: Option<Point>,
```

Comment:
It turns out that two of these are needed - one for the main grid and one for the alt grid (usually used by fullscreen apps). It might be easiest to just keep a 2 element array `[Point; 2]` and ditch the `Option`.